local/local.go

```diff
@@ -242,6 +242,23 @@ func (i *Image) SetLabel(key, val string) error {
 	return nil
 }
 
+func (i *Image) SetOS(osVal string) error {
+	if osVal != i.inspect.Os {
+		return fmt.Errorf(`invalid os: must match the daemon: "%s"`, i.inspect.Os)
+	}
+	return nil
```

Comment:
I changed this to not set anything, and instead compare against the current value of `i.inspect.Os` which is already the daemon's default from `defaultInspect`. If we ever get a multi-OS daemon, we'll have to revisit of course.